index.js

```diff
@@ -48,7 +56,7 @@ function decode(buffer) {
   } else {
     const elements = [];
 
-    while (!elementLength || bytesRead - 2 < elementLength) {
+    while (elementLength == undefined || bytesRead - 2 < elementLength) {
```

Comment:
What's the reason for this change?

Reply:
As far as I can remember it's to separate zero-length elements from indefinite length elements.